deeppavlov/models/kbqa/entity_detection_parser.py

```diff
@@ -13,9 +13,11 @@
 # limitations under the License.
```

Comment:
The full copyright and license should be added.

Reply:
I copied copyright and license strings from other component files in our library.